ush/config_defaults.yaml

```
@@ -460,12 +609,16 @@ workflow:
   #
   # CONSTANTS_FN:
   # Name of the file containing definitions of various mathematical, physical, 
-  # and SRW App contants.
+  # and SRW App contents.
```

I believe this should be constants.

```suggestion
  # and SRW App constants.
```

---

Fixed